internal/cli/cmd/dashboard/grafana_direct.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright (C) 2022-2023 ApeCloud Co., Ltd
+
+This file is part of KubeBlocks project
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+package dashboard
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/apecloud/kubeblocks/internal/cli/util"
+	"github.com/spf13/cobra"
+)
+
+var (
+	clusterType string
+	// Todo: availableTypes is hard code, better to do a dynamic query but where the source from?
+	availableTypes = [...]string{
+		"apecloud-mysql",
+		"cadvisor",
+		"jmx",
+		"kafka",
+		"mongodb",
+		"node",
+		"postgresql",
+		"redis",
+		"weaviate",
+	}
+)
+
+func addCharacterFlag(cmd *cobra.Command, dashboardType *string) {
+	cmd.Flags().StringVar(dashboardType, "type", "", "the cluster type opened directly in dashboard. eg 'apecloud-mysql'")
```

Comment:
I prefer to use `cluster-type`

Reply:
There are many other dashboard types, such `cadvisor|jmx|node`, cluster type is not suitable for these type.

Reply:
> There are many other dashboard types, such `cadvisor|jmx|node`, cluster type is not suitable for these type.

I have remove the `--flag` and use the second args as the dashboard types name. More infos cloud find above.